src/main/java/net/wasdev/wlp/common/plugins/util/DevUtil.java

```diff
@@ -420,8 +420,12 @@ public void copyFile(File fileChanged, File srcDir, File targetDir) throws IOExc
                 fileChanged.getAbsolutePath().indexOf(srcDir.getAbsolutePath()) + srcDir.getAbsolutePath().length());
 
         File targetResource = new File(targetDir.getAbsolutePath() + relPath);
-        FileUtils.copyFile(fileChanged, targetResource);
-        info("Copied file: " + fileChanged.getAbsolutePath() + " to: " + targetResource.getAbsolutePath());
+        try {
+            FileUtils.copyFile(fileChanged, targetResource);
+            info("Copied file: " + fileChanged.getAbsolutePath() + " to: " + targetResource.getAbsolutePath());
+        } catch (Exception ex) {
```

Comment:
Also catch FileNotFoundException, in which case just log as debug.
When catching any other Exception, log as error and include the exception message.

Reply:
@ericglau It now handles FileNotFound exception first.